sdk/js/src/service/claim.js

```diff
@@ -46,24 +46,25 @@ class DeviceClaim {
 
   /**
    * Unclaim an end device.
+   * The Device Claiming Server fetches the device EUIs (if present) from the Identity Server.
    *
    * @param {string} applicationId - The Application ID.
    * @param {string} deviceId - The Device ID.
-   * @param {Array} devEui - The Device dev_eui.
-   * @param {object} joinEui - The Device join_eui.
    * @returns {object} - An empty object on successful requests, an error otherwise.
    */
-  async unclaim(applicationId, deviceId, devEui, joinEui) {
+  async unclaim(applicationId, deviceId) {
     const params = {
       routeParams: {
         'application_ids.application_id': applicationId,
         device_id: deviceId,
       },
     }
 
-    const response = await this._api.EndDeviceClaimingServer.Unclaim(params, {
-      dev_eui: devEui,
-      join_eui: joinEui,
+    const response = await this._api.EndDeviceClaimingServer.Unclaim({
```

Comment:
Why not just use `params` ?

Reply:
I believe these are equivalent, so I opt for lesser lines of code.